engine/menu/frame_type_dialog.asm

```diff
@@ -0,0 +1,43 @@
+INCLUDE "constants.asm"
+
+SECTION "engine/menu/frame_type_dialog.asm", ROMX
+
+FrameTypeDialog:
+	ld hl, .MenuHeader
+	call LoadMenuHeader
+	ld a, [wActiveFrame]
+	inc a
+	ld [wMenuCursorBuffer], a
+	call VerticalMenu
+	jr c, .close
+	ld a, [wMenuCursorY]
+	dec a
+	ld [wActiveFrame], a
+	push de
+	ld de, 3 ; SFX_MENU
```

Comment:
The commented constant is defined in #87, will turn it into the correct one once that's merged in